f2/src/components/formik/datePicker/index.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import { Form, Row, Col, Container } from 'react-bootstrap'
+import { Trans } from '@lingui/macro'
+import { Field } from 'formik'
+
+const DateEntry = ({ field, form, ...props }) => {
+  return (
+    <Col sm="auto" className="date-picker">
+      <Form.Group controlId={props.id}>
+        <Form.Label className="date-label">
+          {<Trans id="whenDidItStart.startDay" />}
+        </Form.Label>
+        <Form.Control
+          className="day-month date-input"
+          type="input"
```

Comment:
Is there an 'input' type in HTML? You might mean 'text'
type='text'

Based on react-bootstrap API, as by default is input, then we can specify type as text, email, password etc

Reply:
Corrected.